client/css/analysis.css

```diff
@@ -0,0 +1,43 @@
+:root {
+    /* the higher the number, the less filled */
+  --percent: 8;
+}
+
+.item {
+    display: block;
+}
+
+.item h2 {
+    text-align:center;
+    width: 100%;
+}
+
+svg {
+   -webkit-transform: rotate(-90deg);
+    transform: rotate(-90deg);
+    display: block; /* svg is auto display-inline*/
+    margin: auto;
+    overflow: visible;
+}
+
+.circle_animation {
+  stroke-dasharray: 440; /* circumference of the circle */
+  stroke-dashoffset: 440;
+}
+
+.donut .circle_animation {
+    -webkit-animation: donut 1s ease-out forwards;
+    animation: donut 1s ease-out forwards;
+}
+
+@-webkit-keyframes donut {
+  to {
+    stroke-dashoffset: var(--percent); /* half filled is 220*/
+  }
+}
+
+@keyframes donut {
+  to {
+    stroke-dashoffset: var(--percent);
+  }
+}
```

Comment:
If you do the animation using `@keyframes`, then it will not be very easy for you to control the percentage from JS. Maybe use the `transition` property instead?

Reply:
Why would it not be easy to control if I just change the variable name for each chart?

Reply:
`@keyframes` won't automatically re-run when you change the value of the `--percent` variable, but a transition will

Reply:
The charts won't ever have to re-run. The page will auto reload for every new song and will create new charts with values.

Reply:
Another issue with not using keyframes is that the animation doesn't start from 0. I want it to be able to start empty and then fill and its easier to use keyframes with that rather than having to js functions @laptou